setup.py

```diff
@@ -74,7 +74,8 @@ def configuration(parent_package='', top_path=None):
 
 
 if __name__ == '__main__':
-    from numpy.distutils.core import setup, Extension
+    #from numpy.distutils.core import setup, Extension
+    from setuptools import setup, Extension
```

Comment:
I know see that about year ago the setup.py has started to retire the configuration feature (by using Extension from the main setup.py). I am not happy with it as I can foresee issues for some users. For instance, libtiff scripts are not installed without the configuration feature. I'd like to undo changes that ignore the configuration feature.

However, setup can still support wheel using the following method: when setup.py is called via wheel, it seems that bdist_wheel should be in sys.argv. When this is the case, the setup.py should use setuptools.setup. Otherwise, it should use numpy.distutils.core.setup.
Can you check what is the content of sys.argv when using wheel? And then apply the above method to setup.py.